### PR TITLE
Update to Microsoft.CodeAnalysis.Testing 1.0.1-beta1.19554.4

### PR DESCRIPTION
**Pull request by sharwell**

### CODE REVIEW

File: src/Microsoft.NetCore.Analyzers/Core/Runtime/SerializationRulesDiagnosticAnalyzer.cs [hunk 1]
```diff
@@ -128,7 +128,8 @@ public override void Initialize(AnalysisContext analysisContext)
                         return;
                     }
 
-                    var isNetStandardAssembly = context.Compilation.ReferencedAssemblyNames.Any(identity => string.Equals(identity.Name, "netstandard", StringComparison.OrdinalIgnoreCase));
+                    var systemObjectSymbol = context.Compilation.GetSpecialType(SpecialType.System_Object);
+                    var isNetStandardAssembly = systemObjectSymbol.ContainingAssembly.Name != "mscorlib";
```
Comment (sharwell) [inline src/Microsoft.NetCore.Analyzers/Core/Runtime/SerializationRulesDiagnosticAnalyzer.cs]: 📝 This recently came up in the context of another rule.

Comment (mavasani) [inline src/Microsoft.NetCore.Analyzers/Core/Runtime/SerializationRulesDiagnosticAnalyzer.cs]: Can we make this an extension method on Compilation? Can be done in a separate PR...

Comment (sharwell) [reply src/Microsoft.NetCore.Analyzers/Core/Runtime/SerializationRulesDiagnosticAnalyzer.cs]: ➡️ #3017

Comment (sharwell) [reply src/Microsoft.NetCore.Analyzers/Core/Runtime/SerializationRulesDiagnosticAnalyzer.cs]: Heh, I commented on this right before you did 👍

**Pull request merged**